lib/project_types/script/layers/infrastructure/assemblyscript_task_runner.rb

```diff
@@ -38,6 +39,16 @@ def dependencies_installed?
           true
         end
 
+        def metadata
```

Comment:
(Nothing for you to change @freynca, just wanted to point this out for the experience team) Since the metadata is generated by the AssemblyScript toolchain, this is a case where AssemblyScript specifically has to read from the FS and create the domain object. We read the wasm from the FS in this file too and I'm sure there will be more use cases later on. There might be a better abstraction where languages can use the repository pattern, so maybe this is something to consider with some of our cli redesigns 🤔